pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -761,6 +767,7 @@ def get_treeinfo_serialized(self):
             # Reset build_timestamp so Pulp will fetch all the addons during the next sync
             treeinfo_serialized['distribution_tree']['build_timestamp'] = 0
             return treeinfo_serialized
+        assert False, "Failed to find treeinfo, something went wrong"
```

Comment:
can we remove this line and open a separate pr for treeinfo related stuff? it does not seem to be related to the changes.

Reply:
@ipanova Do you want the CI fix in a separate PR also?